solutions/fully-configurable/outputs.tf

```diff
@@ -0,0 +1,13 @@
+########################################################################################################################
+# Outputs
+########################################################################################################################
+
+output "cluster_name" {
+  value       = module.ocp_base.cluster_name
+  description = "The name of the provisioned cluster."
+}
+
+output "cluster_id" {
+  value       = module.ocp_base.cluster_id
+  description = "The id of the provisioned cluster."
+}
```

Comment:
I am sure we have more output than this?

Reply:
Yes added.